test/integration/setup.ts

```diff
@@ -16,10 +16,7 @@ export async function suiteClientSetup(this: Mocha.Context): Promise<void> {
   // two times to give time to server to send `ledgerClosed` event
   // so getLedgerVersion will return right value
```

Comment:
Probably should update this comment (`getLedgerVersion` is not the right name at least)